lib/mixlib/install/backend/base.rb

```diff
@@ -0,0 +1,20 @@
+module Mixlib
+  class Install
+    class Backend
+      class Base
```

Comment:
Why? I would leave this to the future. Introduction of well documented and defined APIs for `mixlib-install` should be done in one go. Let's not do this for now.